src/core/load-fragment.c

```diff
@@ -2725,17 +2726,18 @@ int config_parse_syscall_filter(
                         c->syscall_whitelist = true;
 
                         /* Accept default syscalls if we are on a whitelist */
-                        r = syscall_filter_parse_one(unit, filename, line, c, false, "@default", false);
+                        r = syscall_filter_parse_one(unit, filename, line, c, false, "@default", false, -1);
```

Comment:
Will the new -1 error conflict with any previous behavior or assumptions ? SystemCallErrorNumber= ?

Reply:
No, negative value means use errno specified by `SystemCallErrorNumber=`.